index.html

```diff
@@ -32,7 +32,8 @@
           <header>
             <h1 class="Name">Javed Shahbaz</h1>
           </header>
-          <p class="headLine">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+          <p class="headLine">Hello! I am a software developer!
+            I can help you build a product, feature or website. Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
```

Comment:
- [ ] The Footer social media icons are not well aligned, please fix this

Your design
![image](https://user-images.githubusercontent.com/84156714/221374520-349a7501-8c16-429c-bee4-1bce7ba965f1.png)

Figma design
![image](https://user-images.githubusercontent.com/84156714/221374553-353dbd3f-39bd-401c-a9aa-07a8a9cb0cdd.png)

Reply:
Fixed.